web/src/components/RemoteContent.tsx

```diff
@@ -145,10 +149,25 @@ const RemoteContent = ({
         node.addEventListener('click', handleClick)
       }
     })
+
+    // iframe handling - remove not allowed iframe src from DOM and add tracking parameter
+    const iframes = sandBoxRef.current.getElementsByTagName('iframe')
+    Array.from(iframes).forEach((node: HTMLIFrameElement) => {
+      if (buildConfig().allowedIframeSources.some(el => node.src.indexOf(el) > 0) && sandBoxRef.current) {
+        var url = new URL(node.src)
+        url.searchParams.append('dnt', '1')
+        node.setAttribute('src', url.href)
+      } else if (sandBoxRef.current) {
```

Comment:
```suggestion
      } else {
```

Reply:
had to declare a const for the `sandBofRef.current` to avoid this check